src/Agent/NewRelic/Agent/Core/DataTransport/DataStreamingService.cs

```diff
@@ -544,8 +550,9 @@ private bool CreateChannel(CancellationToken cancellationToken)
                 }
                 catch (Exception ex)
                 {
+                    // TODO: I don't think this is actually a response error...
+                    //RecordResponseError();
```

Comment:
Did you mean to include changes like this with the PR to make the integration tests more stable?

Reply:
The call to `_grpcWrapper.CreateChannel` in the legacy library triggers a connection attempt which can result in networking errors or receiving a response from the server (such as grpc unavailable). So I don't think that we want to remove this code.

Reply:
This has been re-added (along with the others I disabled for science)